tests/parser/features/test_logging.py

```diff
@@ -182,7 +187,53 @@ def foo(arg1: bytes <= 29, arg2: bytes <= 31):
     # # Event abi is created correctly
     assert c.translator.event_data[event_id] == {'types': ['bytes4', 'bytes29', 'bytes31'], 'name': 'MyLog', 'names': ['arg1', 'arg2', 'arg3'], 'indexed': [True, True, False], 'anonymous': False}
     # Event is decoded correctly
-    assert c.translator.decode_event(logs.topics, logs.data) ==  {'arg1': b'bar\x00', 'arg2': bytes_helper('bar', 29), 'arg3': bytes_helper('foo', 31), '_event_type': b'MyLog'}
+    assert c.translator.decode_event(logs.topics, logs.data) ==  {'arg1': b'bar\x00', 'arg2': bytes_helper('bar', 29), 'arg3': bytes_helper('', 28) + b'foo', '_event_type': b'MyLog'}
```

Comment:
Add a `#TODO` comment to remember to fix this once you figure out the issue with the padding

Reply:
@fubuloubu I didn't wanna add a TODO so I just figured out the issues with padding 👍